### PR TITLE
Add org-mode publish keybindings to localleader

### PR DESCRIPTION
**Pull request by snuffop**

### CODE REVIEW

File: modules/lang/org/config.el [hunk 1]
```diff
@@ -735,6 +735,13 @@ between the two."
          "s" #'org-store-link
          "S" #'org-insert-last-stored-link
          "t" #'org-toggle-link-display)
+        (:prefix ("P" . "publish")
+         "a" #'org-publish-all
+         "f" #'org-publish-current-file
+         "p" #'org-publish
+         "P" #'org-publish-current-project
+         "s" #'org-publish-sitemap
+         )
```
Comment (hlissner) [inline modules/lang/org/config.el:744]: Please merge the dangling `)` with the previous line.

Comment (snuffop) [reply modules/lang/org/config.el]: done

**Comment by hlissner** [comment]
Would you mind rebasing this PR onto `develop`. A lot of noise snuck into its commit history.

**Comment by snuffop** [comment]
Remove this..   CRUFT from a rebase.

**Comment by snuffop** [comment]
Created https://github.com/hlissner/doom-emacs/pull/3598

**Pull request closed without merging**